src/sql/workbench/contrib/notebook/browser/utils.ts

```diff
@@ -44,3 +44,18 @@ export function highlightSelectedText(): void {
 	}
 }
 
+export function getEncodedLinkUrl(linkUrl: string): string {
```

Comment:
I think it'd be better to keep all link encoding/decoding logic in one place - so the handler class seems like a good place to do that. 

So for now to help ensure that I'd actually suggest that we move this function into the link handler and take away the parameter, instead just using the URL that was passed into the constructor.